product_configurator_wizard/wizard/product_configurator.py

```diff
@@ -648,7 +654,10 @@ def write(self, vals):
                     attr_id: field_val
                 })
             elif attr_line.custom:
-                val = vals[custom_field_name]
+                # For non-binary fields, a custom field value may have been entered which
+                # is 0 or some other way False, and this will not be in the dictionary of
+                # values, so need to assume that if not passed it is a "False" value desired.
+                val = vals.get(custom_field_name, False)
```

Comment:
Ahh, yes, but generally in Odoo when you want to set a field value to a false value, they use False and not None, hence why I used the alternate fallback value.